.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
       REQUIRES_JEKYLL_CI: ${{secrets.REQUIRES_JEKYLL_CI}}
       REQUIRES_CODECOV: ${{secrets.REQUIRES_CODECOV}}
+      PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
Maybe this action should be split into 2 actions because I think this value will be only available for the pull requests and this action is executed for push and pull_request events:

```
on:
  push:
    branches: master
  pull_request:
```